sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/GeneralScalarExpression.java

```diff
@@ -148,6 +148,42 @@
  *    <li>Since version: 3.3.0</li>
  *   </ul>
  *  </li>
+ *  <li>Name: <code>SUBSTRING</code>
+ *   <ul>
+ *    <li>SQL semantic: <code>SUBSTRING(expr1, expr2, expr3)</code></li>
+ *    <li>Since version: 3.3.0</li>
```

Comment:
@cloud-fan Should it is 3.4.0 ?

Reply:
yea, this is probably too late for spark 3.3